### PR TITLE
Working

### PR DESCRIPTION
**Pull request by BSGreaves**
@drdreed Sandwich Maker ready for feedback.

I deviated from the letter (but not the spirit) of the assignment in once instance:
"Each IIFE should expose, in its public interface, a method named add{ingredient} (e.g. addMeat or addVeggies) that accepts a single argument. That argument's value should be the ingredient selected by the user."
My "add ingredients" functions actually take 2 inputs - the ingredient name,  and also whether the user checked or unchecked the box.  That way, it actively updates the final price whenever a checkbox is checked or unchecked. It seemed easier than adding a separate "remove ingredient" function in each IIFE augmenter (which is not included in the instructions).
Thanks

### CODE REVIEW

File: js/bread.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	};
+
+	oldSandwichMaker.getBreadPrices = function() {
+		return breadPrices;
+	}
+
+	oldSandwichMaker.addBread = function(id, checked) { //either adds or subtracts from the final price by taking the id of the option and getting its price
+		if (checked === true) { //"checked" is e.target.checked passed in from requestToppingPrice
+			SandwichMaker.adjustPrice("add", breadPrices[id]); //adjustPrice needs to know whether or not to add or subtract the topping price from the final price
+		}
```
Comment (DevDreed) [inline js/bread.js]: You could do an else here.  Boolean's can only have two values.  So its either true or false.

File: js/cheese.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	};
+
+	oldSandwichMaker.getCheesePrices = function() {
+		return cheesePrices;
+	}
+
+	oldSandwichMaker.addCheese = function(id, checked) {
+		if (checked === true) {
+			SandwichMaker.adjustPrice("add", cheesePrices[id]);
+		}
```
Comment (DevDreed) [inline js/cheese.js]: Same thing here.  Could use an else statement.

File: js/condiments.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+	};
+
+	oldSandwichMaker.getCondimentPrices = function() {
+		return condimentPrices;
+	}
+
+	oldSandwichMaker.addCondiment = function(id, checked) {
+		if (checked === true) {
+			SandwichMaker.adjustPrice("add", condimentPrices[id]);
+		}
```
Comment (DevDreed) [inline js/condiments.js]: Could use an else statement here as well.

File: js/meats.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+	};
+
+	oldSandwichMaker.getMeatPrices = function() {
+		return meatPrices;
+	}
+
+	oldSandwichMaker.addMeat = function(id, checked) {
+		if (checked === true) {
+			SandwichMaker.adjustPrice("add", meatPrices[id]);
+		}
```
Comment (DevDreed) [inline js/meats.js]: Else statement here as well.

File: js/veggies.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+	};
+
+	oldSandwichMaker.getVeggiePrices = function() {
+		return veggiePrices;
+	}
+
+	oldSandwichMaker.addVeggie = function(id, checked) {
+		if (checked === true) {
+			SandwichMaker.adjustPrice("add", veggiePrices[id]);
+		}
```
Comment (DevDreed) [inline js/veggies.js]: And again and else statement.

**Review by DevDreed — CHANGES_REQUESTED** [review]
Just a couple of changes to clean up the code base.  Looking good though.

**Review by DevDreed — APPROVED** [review]
👍

**Comment by BSGreaves** [comment]
Good catches, thanks! Changes made.

**Pull request merged**